src/python/install.sh

```diff
@@ -449,12 +453,17 @@ fi
 
 # Install JupyterLab if needed
 if [ "${INSTALL_JUPYTERLAB}" = "true" ]; then
-    install_user_package jupyterlab
+    if [ ! -z "${PYTHON_SRC}" ]; then
```

Comment:
Another approach here is a guard condition:

```bash
if [ -z "${PYTHON_SRC}" ]; then
    echo "(!) Could not install JupyterLab. Python not found."
    exit 1
fi

install_user_package jupyterlab

# Configure JupyterLab if needed
```

This keeps the error close to the condition and saves us some indenting